students/marta_herezo/lesson_01/lesson_01/Area_of_right_angled_triangle.py

```diff
@@ -0,0 +1,13 @@
+# enter the length of the base
+print('enter length of the base')
+c = int(input())
+
+# enter the height
+print('enter the height')
+h = int(input())
+
+P = 0.5 * c * h
```

Comment:
Try to name variables properly:
P - should be lowwer
P - should be 'area', nobody knows what 'p' does it mean :)